File: app/src/main/java/com/gmail/fattazzo/formula1world/domain/F1DriverStandings.java
package com.gmail.fattazzo.formula1world.domain;

/**
 * @author fattazzo
 *         <p/>
 *         date: 05/06/17
 */
public class F1DriverStandings {

    public int position;

    public String positionText;

    public float points;

    public int wins;

    public F1Driver driver;

    public F1Constructor constructor;

    @Override
    public String toString() {
        return "F1DriverStandings{" +
                "position=" + position +
                ", positionText='" + positionText + '\'' +
                ", points=" + points +
                ", wins=" + wins +
                ", driver=" + driver +
                ", constructor=" + constructor +
                '}';
    }
}

File: app/src/main/kotlin/com/gmail/fattazzo/formula1world/domain/F1DriverStandings.kt
package com.gmail.fattazzo.formula1world.domain

/**
 * @author fattazzo
 *
 *
 * date: 05/06/17
 */
class F1DriverStandings {

    var position: Int = 0

    var positionText: String? = null

    var points: Float = 0.toFloat()

    var wins: Int = 0

    var driver: F1Driver? = null

    var constructor: F1Constructor? = null

    override fun toString(): String {
        return "F1DriverStandings{" +
                "position=" + position +
                ", positionText='" + positionText + '\'' +
                ", points=" + points +
                ", wins=" + wins +
                ", driver=" + driver +
                ", constructor=" + constructor +
                '}'
    }
}
